spec/features/contestants/index_spec.rb

```diff
@@ -0,0 +1,45 @@
+require "rails_helper"
+
+RSpec.describe "the contestants index" do
+  before :each do
+    @jay = Contestant.create(name: "Jay McCarroll", age: 40, hometown: "LA", years_of_experience: 13)
+    @gretchen = Contestant.create(name: "Gretchen Jones", age: 36, hometown: "NYC", years_of_experience: 12)
+    @kentaro = Contestant.create(name: "Kentaro Kameyama", age: 30, hometown: "Boston", years_of_experience: 8)
+    @erin = Contestant.create(name: "Erin Robertson", age: 44, hometown: "Denver", years_of_experience: 15)
+
+    @recycled_material_challenge = Challenge.create(theme: "Recycled Material", project_budget: 1000)
+    @furniture_challenge = Challenge.create(theme: "Apartment Furnishings", project_budget: 1000)
+
+    @news_chic = @recycled_material_challenge.projects.create(name: "News Chic", material: "Newspaper")
+    @boardfit = @recycled_material_challenge.projects.create(name: "Boardfit", material: "Cardboard Boxes")
+    @upholstery_tux = @furniture_challenge.projects.create(name: "Upholstery Tuxedo", material: "Couch")
+    @lit_fit = @furniture_challenge.projects.create(name: "Litfit", material: "Lamp")
+
+    @jay_project = ContestantProject.create(contestant_id: @jay.id, project_id: @news_chic.id)
+    @gretchen_project_1 = ContestantProject.create(contestant_id: @gretchen.id, project_id: @news_chic.id)
+    @gretchen_project_2 = ContestantProject.create(contestant_id: @gretchen.id, project_id: @upholstery_tux.id)
+    @kentaro_project_1 = ContestantProject.create(contestant_id: @kentaro.id, project_id: @upholstery_tux.id)
+    @kentaro_project_2 = ContestantProject.create(contestant_id: @kentaro.id, project_id: @boardfit.id)
+    @erin_project = ContestantProject.create(contestant_id: @erin.id, project_id: @boardfit.id)
+  end
+
+  it "shows a list of all contestant names" do
+    visit "/contestants"
+
+    expect(page).to have_content("All Contestants")
+    expect(page).to have_content(@jay.name)
+    expect(page).to have_content(@gretchen.name)
+    expect(page).to have_content(@kentaro.name)
+    expect(page).to have_content(@erin.name)
+  end
+
+  it "shows projects that contestants have been on" do
+    visit "/contestants"
+
+    expect(page).to have_content("Projects")
```

Comment:
Again I would use `within` blocks here to ensure that for each contestant it is truly displaying their projects. Something like:
```ruby
within "#contestant_#{@jay.id}" do
   expect(page).to have_content("News Chic")
   expect(page).to_not have_content("Upholstery Tuxedo")
   expect(page).to_not have_content("Boardfit")
   expect(page).to_not have_content("Litfit")
end
```
Then in your view were you are iterating over `@contestants` you would add something like:
```ruby
<% @contestants.each do |contestant| %>
    <li id="contestant_<%= contestant.id %>"><%= contestant.name %> <br/>
       Projects: <%= contestant.projects_list %>
    </li>
<% end %>
```